mne/preprocessing/maxwell.py

```diff
@@ -0,0 +1,507 @@
+# Authors: Mark Wronkiewicz <wronk.mark@gmail.com>
+#          Jussi Nurminen <jnu@iki.fi>
+
+# This code was adapted and relicensed (with BSD form) with permission from
+# Jussi Nurminen
+
+# License: BSD (3-clause)
+
+# Equation numbers refer to Taulu and Kajola, 2005. 'Presentation of
+# electromagnetic multichannel data: The signal space separation method,' which
+# can be found for free online.
+
+# Note, there are an absurd number of different possible notations for
+# spherical coordinates, which confounds the notation for spherical harmonics.
+# Here, we purposefully stay away from shorthand notation in both and use
+# explicit terms (like 'azimuth' and 'polar') to avoid confusion.
+
+
+from __future__ import division
+from os import path as op
+import numpy as np
+from scipy.special import lpmv
+from scipy.linalg import pinv
+from scipy.misc import factorial as fact
+from ..utils import logger
```

Comment:
any function that uses `logger` will need to be decorated by `@verbose`, so import that here and use it where you need it. Also by convention we separate the absolute other-package imports from the `mne` relative imports by an empty line